spec/eofv0_verkle.md

```diff
@@ -111,50 +111,34 @@ Alternate option is instead of encoding all valid `JUMPDEST` locations, to only
 By invalid `JUMPDEST` we mean a `0x5b` byte in any pushdata.
 
 This is beneficial if our assumption is correct that most contracts only contain a limited number
-of offending cases. Our initial analysis suggests this is the case, e.g. Uniswap router has 9 cases,
-one of the Arbitrum validator contracts has 6 cases.
+of offending cases. Our initial analysis of the top 1000 used bytecodes suggests this is the case:
+only 0.07% of bytecode bytes are invalid jumpdests.
 
-Since Solidity contracts have a trailing metadata, which contains a Keccak-256 (32-byte) hash of the
```

Comment:
Don't we went to keep this trivia?

Reply:
We have data now so this estimation is pointless. I actually checked and probability from all dataset is indeed ~12%.

Reply:
This actually wasn't correct. The contract hash is not in the `PUSH32` therefore doesn't count towards invalid jumpdests.